src/pages/Sketch/components/Canvas.module.css

```diff
@@ -1,8 +1,8 @@
 .canvas, .dark_canvas {
-    border: 2.5px solid blue;
     border-radius: 11px;
```

Comment:
remove the space present here too

Reply:
Hey did you mean to remove border-radius attribute too ?

Reply:
there's extra space as you have deleted just move that bit little up.